draft-fossati-tls-attestation.md

```diff
@@ -262,23 +296,21 @@ attestation, and the other uses the background check model.
 
 ## Handshake Overview {#handshake-overview}
 
-The handshake defined here is analogous to certificate-based authentication in a regular TLS handshake. Instead of the certificate's private key, we use
-the TIK identity key. This key is attested, with attestation being carried
-by the Certificate message. Following that, the peer being attested proves possession of the private key using the CertificateVerify message.
+The handshake defined here is analogous to certificate-based authentication in a regular TLS handshake.
+We use the TLS Identity Key (TIK) which is either a stand-alone key or is identical
+to the certificate's private key (see {{tik}}).
+This key is attested, with attestation being carried
+in the Certificate message. Following that, the peer being attested proves possession of the private key using the CertificateVerify message.
 
 Depending on the use case, the protocol supports peer authentication
 using attestation only, or using both attestation and a regular public
 key certificate.
 
 The current version of the document assumes the KAT/PAT construct of 
 {{I-D.bft-rats-kat}}. Not all platforms support this model, and a document
-that defines private key attestation for use in TLS Attestation as defined here, must specify:
-
-* The format and the lifetime of TIK (e.g. an ephemeral, per session TIK vs.
-a long lived one).
-* How the key is attested using a structure carried by the
+that defines private key attestation for use in TLS Attestation as defined here, must specify
```

Comment:
Ditto, as my previous comment (i.e., s/must/MUST/).

Reply:
I try to avoid normative language when it applies to future documents rather than implementations. Maybe a matter of taste.